web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -1049,6 +1049,239 @@ $.extend(fixmystreet.set_up, {
         var set_map_state = true;
         fixmystreet.back_to_reports_list(e, report_list_url, map_state, set_map_state);
     });
+  },
+
+  expandable_list_items: function(){
+      // All of the .off/.on event binding here is intended to allow us
+      // to call fixmystreet.set_up.expandable_list_items() multiple times
+      // (eg: after report list items are loaded by ajax) without the
+      // effects stacking up on top of each other.
+      $(document).off('click.toggle-expansion').on('click.toggle-expansion', '.js-toggle-expansion', function(e){
+          e.preventDefault(); // eg: prevent button from submitting parent form
+          var $li = $(this).closest('li');
+          if ( $li.is('.expanded') ) {
+              $li.removeClass('expanded');
+              $(this).text(translation_strings.read_more);
+          } else {
+              $li.addClass('expanded');
+              $(this).text(translation_strings.read_less);
+          }
+      });
+
+      var $list_items = $('.js-toggle-expansion').closest('li');
+      $list_items.off('click.toggle-expansion').on('click.toggle-expansion', function(e){
+          var $li = $(this);
+          // Ignore list items that are already expanded.
+          if ( ! $li.is('.expanded') ) {
+              // Ignore clicks on action buttons (clicks on the
+              // .js-toggle-expansion button will be handled by
+              // the more specific handler above).
+              if ( ! $(e.target).is('.item-list__item--expandable__actions *') ) {
+                  e.preventDefault();
+                  $li.addClass('expanded');
+                  $li.find('.js-toggle-expansion').text(translation_strings.read_less);
+              }
+          }
+      });
+  },
+
+  manage_duplicates: function() {
+    // Can skip all this if there’s no duplicates container on the page.
```

Comment:
Lot of extra JS though, in that case. Can we perhaps have this in a separate JS file, that is only included then if that template also is? Might be easier/better than separating out staff/non-staff, dunno.